### PR TITLE
Fix 5e.tools incompatibilities

### PR DESCRIPTION
**Pull request by hogart**
Addresses #89

The export is still not ideal, but most of the fields are transferred properly, and the creatures are actually visible in 5.tools' homebrew editor.

Also adds minor non-breaking feature to allow the server to run on different port.

### CODE REVIEW

File: server/controllers/users.js [hunk 1]
```diff
@@ -9,17 +9,8 @@ var creatures = require("../controllers/creatures");
...
-        to: emailAddress,
-        subject: 'Welcome to Critter DB',
-        text: 'Thanks for joining Critter DB! I hope it helps you with all your critter management needs. Please send any feedback to me at haswellrd@gmail.com or on Github at https://github.com/haswellr/CritterDB.\n\nThanks,\nRyan'
-    };
-    try {
-        emailController.sendEmail(mailOptions);
-    } catch (error) {
-        console.log(error);
-    }
+    return;
```
Comment (haswellr) [inline server/controllers/users.js:12]: I think these changes accidentally slipped in. Otherwise looks good.

**Review by haswellr — CHANGES_REQUESTED** [review]
If you remove the change to the welcome email, I'll push and deploy these. Thanks for helping out!

**Comment by hogart** [comment]
> If you remove the change to the welcome email, I'll push and deploy these. Thanks for helping out!

My bad! Fixed. 
Please let me know once it's deployed, I need to port quite a bestiary:)

**Review by haswellr — APPROVED** [review]

**Comment by haswellr** [comment]
I had to put in a quick change because CritterDB is terribly old and the build process doesn't support the spread operator, but it is deployed now!

**Comment by hogart** [comment]
Thank you very much for the great tool!

**Pull request merged**